src/asmcnc/core_UI/job_go/popups/popup_yetipilot_settings.py

```diff
@@ -128,8 +128,8 @@ def __init__(self, screen_manager, localization, machine, database, yetipilot, v
         sum_of_middle_heights = subtitle_height  + radio_BL_height + body_BL_height
         close_button_BL_height = vertical_BL_height - sum_of_middle_heights
 
-        dropdowns_container_width = 350
-        dropdowns_width = 270
+        dropdowns_container_width = 350 - 20
```

Comment:
Is this intentional? In place of just 330

Reply:
Ah it was so I could keep track of the changes I was making to the background at the same time 😂. Have updated, let me know if you're happy @DennisKiselev 😁